### PR TITLE
add test request to 'ping' yw-website

### PR DESCRIPTION
**Pull request by aniehuser**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: java
 script: mvn test
 jdk:
-  - openjdk7
-  - oraclejdk7
\ No newline at end of file
+  - openjdk10
+  - openjdk11
+  - oraclejdk11
+
```
Comment (clundin55) [inline .travis.yml]: EOF

File: src/main/java/org/yesworkflow/save/HttpSaver.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        client = HttpClients.custom()
+                .build();
+    }
+
+    public void save()
+    {
+        Scanner scanner;
+
+        try {
+            HttpGet httpGet = new HttpGet("http://localhost:8000/save/ping");
```
Comment (clundin55) [inline src/main/java/org/yesworkflow/save/HttpSaver.java]: don't hard code server address. I know you are doing this just for the prototype so I won't block this PR, just noting.

Comment (aniehuser) [reply src/main/java/org/yesworkflow/save/HttpSaver.java]: The next task is adding configuration support to remove the hard coding

**Comment by clundin55** [comment]
You shouldn't add changes after the PR has been approved. Instead add them and make a new PR. Otherwise you'll sneak by me ;)

**Pull request merged**